modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1147,7 +1236,9 @@ public TableImpl table(UUID id) throws NodeStoppingException {
             return CompletableFuture.completedFuture(null);
         }
 
-        var tbl = tablesById.get(id);
+        Map<UUID, TableImpl> tablesById = tablesByIdVv.get().join();
```

Comment:
I am not sure that such calls are NPE-safe.

Reply:
Below the line, I am checking the value to null.